x-pack/plugins/security_solution/public/assistant/get_comments/stream/index.tsx

```diff
@@ -7,6 +7,7 @@
 
 import React, { useEffect, useMemo, useRef } from 'react';
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { useFetchConnectorsQuery } from '../../../detection_engine/rule_management/api/hooks/use_fetch_connectors_query';
```

Comment:
nit: move to `useFetchConnectorsQuery` to `/security_solution/public/common/`

Reply:
Even better would be to add a `featureId` argument to `useFetchConnectorTypesQuery` and pass the `featureId` to `fetchConnectorTypes` to use as the `feature_id` argument. Then we could call like:
```
useFetchConnectorTypesQuery(GenerativeAIForSecurityConnectorFeatureId)
```

Available `featureId`s: https://github.com/elastic/kibana/blob/2759182d496bda79f847e9b671de7da2fe18124d/x-pack/plugins/actions/common/connector_feature_config.ts#L24-L29

Reply:
actually maybe we should use `useLoadActionTypesQuery` from `triggers_actions_ui`. we'd just need to add an arg for `featureId` and default it to `AlertingConnectorFeatureId`. It could use the existing function `loadActionTypes` instead of `fetchConnectorTypes `